config/F16.config

```diff
@@ -8,49 +8,67 @@ process {
     }
     withName: call_sSNV_SomaticSniper {
         cpus = 1
-        memory = 10.GB
+        memory = 1.GB
+        retry_strategy {
+            memory {
+                strategy = 'add'
+                operand = 3.GB
+            }
+        }
     }
     withName: convert_BAM2Pileup_SAMtools {
         cpus = 1
-        memory = 10.GB
+        memory = 1.GB
+        retry_strategy {
+            memory {
+                strategy = 'add'
+                operand = 3.GB
+            }
+        }
     }
     withName: create_IndelCandidate_SAMtools {
         cpus = 1
-        memory = 10.GB
+        memory = 1.GB
+        retry_strategy {
+            memory {
+                strategy = 'add'
+                operand = 3.GB
+            }
+        }
     }
     withName: call_sIndel_Manta {
-        cpus = 16
-        memory = 15.GB
+        cpus = 6
+        memory = 6.GB
         retry_strategy {
             memory {
                 strategy = 'add'
-                operand = 5.GB
+                operand = 3.GB
```

Comment:
There are many unrelated changes in the `FX.config` in this PR that don't match the PR title or SomaticSniper. Maybe we could either incorporate the changes in https://github.com/uclahs-cds/pipeline-call-sSNV/pull/256 or open a separate PR for them?

Reply:
Sorry @tyamaguchi-ucla, those changes were in the other branch, but I had failed to update it here.

Reply:
Looks good to me! Maybe merge this branch into the main branch instead of #256?

Reply:
Either order of merge into main should work, as long as we do: merge one branch into main -> update remaining PR and test -> merge into main